src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonPropertyInfo.cs

```diff
@@ -172,6 +172,30 @@ internal JsonConverter EffectiveConverter
             }
         }
 
+        private JsonObjectCreationHandling? _objectCreationHandling;
+        internal JsonObjectCreationHandling EffectiveObjectCreationHandling { get; private set; }
+
+        /// <summary>
+        /// Gets of sets a value indicating if member will be replaced or populated during deserialization.
+        /// When <see langword="null"/> default value will be used.
```

Comment:
Could you move the second sentence to the remarks section please?

Reply:
done in the next iteration